src/web/routes/configure/configure-router.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpStatusCode } from 'axios';
+import type { NextFunction } from 'express';
+import { Router } from 'express';
+
+import { CONFIGURE_FIGMA_TEAMS_REQUEST_BODY } from './schemas';
+import type {
+	ConfigureFigmaTeamsRequest,
+	ConfigureFigmaTeamsResponse,
+} from './types';
+
+import { assertSchema } from '../../../infrastructure';
+import { configureFigmaTeamUseCase } from '../../../usecases';
+import {
+	authHeaderSymmetricJwtMiddleware,
+	extractUserIdFromHeadersMiddleware,
+} from '../../middleware';
+
+export const configureRouter = Router();
+
+configureRouter.use(
+	extractUserIdFromHeadersMiddleware,
+	authHeaderSymmetricJwtMiddleware,
+);
+
+configureRouter.post(
+	'/team',
```

Comment:
**suggesgtion:** The containing folder and the router are called `configure` but this name can be a too generic and does not provide the context about what we configure here. If we need to add to the system something else configurable down the road, this folder might be not a good place for the new code (since it would contain configurable but _unrelated_ things).

Therefore, consider:

1. Renaming the containing folder from `configure` to `team` to add more context.
2. Renaming the endpoint from `configure/team` to `team/configure`.